src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -127,6 +130,7 @@ static void print_help_messages() {
                               " For example fc:onednn gemm:onednn reduce:ocl do:cpu"
                               " For primitives fc, gemm, do, reduce, concat are supported. Separated by space.");
     message_list.emplace_back("OV_GPU_MaxKernelsPerBatch", "Maximum number of kernels in a batch during compiling kernels");
+    message_list.emplace_back("OV_GPU_DumpIteration", "Dump n-th execution of network, separated by comma.");
```

Comment:
How about use the separate by space? debug_config usuall use sperated by space.

Reply:
Changed

Reply:
Please update help message as well. `separated by comma` -> `separated by space`